src/index.js

```diff
@@ -1,27 +1,34 @@
 // import _ from 'lodash';
 import './style.css';
+import status from './script.js';
 
 const tasks = [
   {
     description: 'wash the dishes',
     completed: false,
-    index: 1,
+    index: 0,
   },
 
   {
     description: 'do the laundry',
     completed: false,
-    index: 2,
+    index: 1,
   },
 
   {
     description: 'wash the car',
     completed: false,
-    index: 3,
+    index: 2,
   },
 ];
 
-tasks.forEach((element) => {
+let taskList = JSON.parse(localStorage.getItem('tasks'));
```

Comment:
- [ ] I am unable to see any change in the local storage variables, before or after refreshing the page. Please take a look at this bug and solve it to comply with the project requirements.

Reply:
Okay, I've arranged the bug and I've uploaded two screenshots to show the changes applied to the local storage before and after a user clicks the checkbox. It also stays the same on reload
![Screenshot1](https://user-images.githubusercontent.com/85113567/135399896-41353135-4f9a-4005-bcd4-e6da6bacade4.jpg)
![Screenshot2](https://user-images.githubusercontent.com/85113567/135399902-1c26c34a-a81e-4497-8b26-4620377c11bf.jpg)
.